plotsearch/models.py

```diff
@@ -172,6 +173,31 @@ def clean(self):
         raise ValidationError(code="no_adding_searchable_targets_after_begins_at")
 
 
+class InformationCheck(NameModel):
+    """
+    In Finnish: Lisätiedon tila
+    """
+
+    # In Finnish: Käsittelijä
+    preparer = models.ForeignKey(
+        User,
+        verbose_name=_("Preparer"),
+        related_name="+",
+        null=True,
+        blank=True,
+        on_delete=models.PROTECT,
+    )
+
+    # In Finnish: Vastauksen osio
+    entry_section = models.ForeignKey(EntrySection, on_delete=models.PROTECT)
+
+    # In Finnish: Tila
+    state = EnumField(enum=InformationState, max_length=30, null=True, blank=True)
```

Comment:
Not sure if it's customary to have a null value as a part of the possible options for an enum field. I would probably not allow null here and add a new "not checked" value into the enum instead, though I'm fine with it being either way.